browser/module.go

```diff
@@ -14,7 +16,10 @@ const version = "0.8.0"
 type (
 	// RootModule is the global module instance that will create module
 	// instances for each VU.
-	RootModule struct{}
+	RootModule struct {
+		muBrowserProcessIDs sync.RWMutex
```

Comment:
Should this be `browserProcessIDsMu` or `browserPIDsMu` instead? I believe we usually use the `mu` to identify the mutex as suffix instead of prefix.

Reply:
Both are valid conventions: prefixed or suffixed. Sure, we can call it `browserPidsMu` (Pid is a convention in Go—stdlib). WDYT?

Reply:
Got it, intuitively I would have thought that by other conventions should be `PID`, but you are right `Pid` is used in stdlib, so LGTM.